definitions/features/redis.rb

```diff
@@ -17,24 +17,12 @@ def config_files
   end
 
   class << self
-    SCL_NAME = 'rh-redis5'.freeze
-
     def etc_prefix
```

Comment:
I debated if this is still needed. In EL9 the config is `/etc/redis/redis.conf` and on Debian it's always been in `/etc/redis/redis.conf` too. Looking at `config_files` it already always probes `/etc/redis` now so perhaps it is redundant.